Dockerfile

```diff
@@ -17,15 +17,27 @@ RUN yarn && yarn task build:server:binary
 
 # We deploy with ubuntu so that devs have a familiar environment.
 FROM ubuntu:18.10
-WORKDIR /root/project
-COPY --from=0 /src/packages/server/cli-linux-x64 /usr/local/bin/code-server
-EXPOSE 8443
-RUN apt-get update && apt-get install -y \
-	openssl \
-	net-tools
-RUN apt-get install -y locales && \
-	locale-gen en_US.UTF-8
+
 # We unfortunately cannot use update-locale because docker will not use the env variables
 # configured in /etc/default/locale so we need to set it manually.
 ENV LANG=en_US.UTF-8
-ENTRYPOINT ["code-server"]
+
+COPY --from=0 /src/packages/server/cli-linux-x64 /usr/local/bin/code-server
+RUN apt-get update && apt-get install -y \
+    sudo \
```

Comment:
Please don't combine this. Minimizing layers in favour of readability is premature optimization.

https://github.com/codercom/code-server/issues/183#issuecomment-472776658

Reply:
By combine I mean the `echo` and `adduser` stuff.

Its fine to install all deps in a single RUN.

Reply:
I don't think it merits its own `RUN`? It's better to have it on one build step already than to force it in another layer when they do the exact same thing. 

On a novice Linux user's perspective, which I also asked from a couple of Linux beginners, this is still readable, since you can tell after `apt` the next step is adding user and changing modes, which most are acquainted at already. Which defeats your "Minimzing layers in favor of readability" point. If you really want unreadable Docker images, look further more into the official library images.

This chain is actually more innocent than the rest of how the Docker library images are done (where compliancy is a must). And to validate my claim this is readable in most beginners' perspective, I asked people to read it and ask if they can read and describe me how everything works (which 94% of the people agreeing this is readable).

Reply:
> It's better to have it on one build step already than to force it in another layer when they do the exact same thing.

What does "the exact same thing"? If it did the "exact same thing", why would you do it twice?

This command has three purposes: installing dependencies, adding a user, and configuring application state. None of those three should be in the same step; they are all distinct, unrelated operations.